internal/operators/handler.go

```diff
@@ -0,0 +1,40 @@
+package operators
```

Comment:
What's the difference between operators handler and operators manager?

Reply:
handler implements REST API interface and uses rest API models, while manager is a layer below and can be used in any context.

Reply:
Can you add comments to each object/module?